js/repl/src/PluginConfig.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+
+import type { PluginConfig } from './types';
+
+const pluginConfigs: Array<PluginConfig> = [
+  {
+    baseUrl: 'https://unpkg.com',
+    label: 'Minify (Babili)',
+    package: 'babili-standalone',
+    version: '0'
+  },
+  {
+    label: 'Prettify',
+    package: 'prettier',
+    version: '1'
+  }
+];
+
+const presetPluginConfigs: Array<PluginConfig> = [
+  /* TODO Add support for preset-env
+  {
+    label: 'env',
+    package: 'babel-preset-env',
+    version: '1'
+  },
+  */
+  {
+    label: 'es2015',
+    package: 'babel-preset-es2015',
+    version: '6'
+  },
+  {
+    label: 'es2015-loose',
+    package: 'babel-preset-es2015-loose',
+    version: '7'
```

Comment:
Even after https://github.com/babel/babel-standalone/pull/90 is merged, `babel-preset-env` will still be a separate package due to how large it is (as it needs all those compat lists to determine which plugins to execute). With all the other presets and plugins, they're small enough (compared to Babel itself) that there's not really an advantage of splitting them out into their own packages.